docs/styleForAPI.css

```diff
@@ -1,189 +0,0 @@
-/* @override http://hudson.openmicroscopy.org.uk/userContent/styleForAPI.css */
-
-/*
- * Style for OMERO API 
- */
-
-html {
-	margin: 10px;
-	padding: 0;
-	background-color: #B3B3E6;
-	font-size: 12px;
-	font-family: Verdana, Arial, Helvetica, sans-serif;
-}
-
-#version-top table.HeaderFooter tbody tr td {
-}
-
-body {
-	margin: 5px 5px 5px 5px;
-	border: 1px solid black;
-	padding: 5px 5px 5px 7px;
-	background-color: #ffffff;
-}
-
-img {
-	border: 0px;
-	padding: 0px;
-	vertical-align: middle;
-}
-
-.HeaderFooter {
-background-color: #B3B3E6;
-padding: 0em 0em 0em 0em;
-white-space: nowrap;
-list-style: none;
-margin: 5px 0px 5px 0px;
-height: auto;
-line-height: normal;
-font-size:12pt;
-}
-
-#version-top table.HeaderFooter tbody tr td table{
-margin: 0;
-}
-
-#version-top p {
-	position: absolute;
-	top: 90px;
-	right: 25px;
-}
-
-#version-top table.HeaderFooter tbody tr td{
-	padding-top: 0;
-	padding-bottom: 0;
-	padding-left: 10px;
-}
-
-.HeaderFooter a:link {
-	color: #000066;
-	text-decoration: none;
-}
-.HeaderFooter a:visited {
-	color: #000066;
-	text-decoration: none;
-}
-
-.HeaderFooter a:hover {
-	color: #000;
-	text-decoration: none;
-}
-
-a:link,a:visited {
-	color: #44448c;
-/*	text-decoration: none;*/
-}
-
-a:hover {
-	color: #000066;
-}
-
-a.Symbol {
-	font-family: "Courier New", Courier, mono;
-	font-weight: bold;
-}
-
-p {
-	text-align: left;
-	padding-left: 20px;
-}
-
-p.Deprecated {
-	border-left: 3em solid red;
-	color: #c50038;
-}
-
-pre {
-	padding-left: 40px;
-	font-weight: bold;
-	line-height: 1.3;
-	font-family: "Courier New", Courier, mono;
-}
-
-span.Synopsis {
-	font: bold larger "Courier New", Courier, mono;
-	color: #656565;
-}
-
-dl {
-	padding-left: 20px;
-}
-
-dt.Symbol {
-	margin-top: 1px;
-	font-size: larger;
-}
-
-h1 {
-	color: #000066;
-	font-size: 24px;
-	font-weight: bold;
-	background-color: #f7f7f7;
-}
-
-h1.Symbol {
-	padding-left: 5px;
-	font: bold 24px "Courier New", Courier, mono;
-	background-color: #FFF;
-}
-
-h2 {
-	color: #000066;
-	font-weight: bold;
-	background-color: #f7f7f7;
-	padding-left: 5px;
-}
-
-h3.Synopsis {
-	padding-left: 20px;
-	font-family: "Courier New", Courier, mono;
-}
-
-hr {
-	opacity: 0.4;
-}
-
-table {
-	margin: 5px;
-}
-
-th {
-	border: 1px solid #aaa; 
-	background-color: #f7f7f7;
-}
-
-td {
-	text-align: left;
-	vertical-align: middle;
-	padding: 2px 2px 2px 20px;
-}
-
-button {
-	display: none;
-	background-color: #e9e9f6;
-	border: thin solid #b7b7e5;
-	font-family: "Helvetica Neue", Arial, Helvetica, Geneva, sans-serif;
-	font-weight: bold;
-}
-
-#SymbolTree {
-	font-size: larger;
-	font-variant: normal;
-	list-style-position: outside;
-}
-
-#banner {
-	background-color: #000066;
-	background-image: url(http://hudson.openmicroscopy.org.uk/userContent/ome_trac_back_top.png);
```

Comment:
http://downloads.openmicroscopy.org/resources/styleForAPI.css has this too, will that also need updating?